docs/HACKING.md

```diff
@@ -126,3 +150,24 @@ Everything is orchestrated by [ci.zig](/src/scripts/ci.zig) script:
 ```console
 ./zig/zig build scripts -- ci --language=go
 ```
+
+## Pull Requests
+
+When submitting pull request, _assign_ a single person to be its reviewer. Unpacking:
+
+* GitHub supports both "assign" and "request review". The difference between them is that "request"
+  is "edge triggered" (it is cleared after a round of review), while "assign" is "level triggered"
+  (it won't go away until the PR is merged or closed). We use "assign", because the reviewer is
+  co-responsible for making sure that the PR doesn't stall, and is eventually completed.
+
+* Only a single person is assigned to any particular pull request, to avoid diffusion of
```

Comment:
@emschwartz I think we should go for this approach for docs PRs as well, as right now it's not clear who's actually blocks the PR from merging. 

It might be challenging to pick the right reviewer for the docs, *but* given that we all want to play support role and help users in slack, I think picking anybody for any document would work actually: we all should now the entirety of our users docs!